2-mandatory/4-people-i-know.js

```diff
@@ -393,7 +395,13 @@ You can see who people's colleagues are by seeing the "colleagues" array in each
 This time, I only want the full names ("<firstname> <lastname>") of my friends who are colleagues of hers.
 */
 
-let friendsWhoAreColleaguesOfStacie = [];
+let friendsWhoAreColleaguesOfStacie = friends
+  .filter((friend) =>
+    friend.colleagues.some(
+      (colleague) => colleague.name === "Stacie Villarreal"
+    )
+  )
+  .map((friend) => `${friend.name.first} ${friend.name.last}`);
```

Comment:
You know, I forgot all about `some` and struggled doing this test myself! Goes to show, no matter how much experience you have, we are all learning! This code is better than mine! Great work Laleh!!!